packages/frontend/src/translations/en.global.json

```diff
@@ -302,11 +304,33 @@
     "confirmLedgerModal": {
         "header": {
             "confirm": "Please confirm the operation on your device...",
-            "processing": "Processing"
+            "processing": "Processing",
+            "success": "accounts have been successfully imported from your Ledger device.",
+            "weFound": "We found ${totalAccounts} accounts secured by your device."
         },
-        "one": "You will need to confirm your sign-in details on your Ledger."
+        "one": "You will need to confirm your sign-in details on your Ledger.",
+        "two": "Follow the instructions on your device to confirm and grant access to each account you'd like to import."
     },
     "connecting": "Connecting",
+    "connectLedger": {
+        "connectLedger": "Connect Ledger",
+        "ledgerConnected": "Ledger Connected",
+        "modal": {
+            "connect": {
+                "header": "Connect to your Ledger device.",
+                "one": "Make sure your Ledger is connected securely, and that the NEAR app is open on your device."
```

Comment:
Can we factor this out to be directly under the `modal` key? If we need to update it in the future it would be better to have it in one place

Reply:
Right, thanks for catching it. That was the thing I forgot to ask @corwinharrell  about - because we have three different modals but the same text on design. I think we should provide some additional info for every modal that might help users to understand the problem. Something like:
 - on error - your device is probably not physically connected or you need to enter a pin
 - on disconnected - your device was probably disconnected physically.
Maybe you can provide some nice text and I will change it?